cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -266,6 +277,50 @@ func main() {
 		controllerCapabilities,
 	)
 
+	var capacityController *capacity.Controller
+	if (*capacityFeatures)[capacity.FeatureCentral] {
+		// TODO: use parameters instead?
```

Comment:
@jsafrane: do you have an opinion whether it is worth making POD_NAME and POD_NAMESPACE real parameters? The advantage is that they become more visible. The disadvantage is that the YAML file becomes more complex because the variables are still needed (we need the env/valueFrom/fieldRef and the parameter which uses it).

Also, is it worth adding a parameter for how far the ownership chain should be walked up? It's not needed for StatefulSet where the desired owner is one level up (current code, hard-coded), but it may be useful for a Deployment->ReplicaSet->Pod scenario, because then the owner should be the Deployment object.

The problem is that in order to retrieve arbitrary parent objects, I need to pull in code like sigs.k8s.io/controller-runtime/pkg/client which can get objects of arbitrary types.

Reply:
I decided to stick with just the env variables.

Reply:
Env. variables are pretty common. For ownership, I can see that using Deployment is far better than using ReplicaSet, however, I don't know how to make it generic and nice. Perhaps a special case for ReplicaSet?

Reply:
I've implemented it using sigs.k8s.io/controller-runtime/pkg/client: that API works with arbitrary object types.

I just pushed that updated code.